content/components/selectpanel.mdx

```diff
@@ -524,29 +413,14 @@ See the [Dialog guidelines](https://primer.style/components/dialog) for more det
 | ------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------- |
 | <Box as="kbd" backgroundColor="canvas.subtle" borderRadius={1} borderColor="border.default" borderWidth={1} borderStyle="solid" py={1} px={2}>ArrowDown</Box> | Navigate through listbox items. Selection follows focus. |
 
-#### Example behavior
-
-<Box display="grid" gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
-  <div>
-    <CustomVideoPlayer
-      width="100%"
-      src="https://user-images.githubusercontent.com/40274682/228391029-44ab995b-7a04-4b85-825e-dc77f90838dd.mp4"
-    />
-    <Caption>
-      Keyboard interaction with a single-select select panel demonstrating selection following focus with navigation via
-      the ArrowDown key.
-    </Caption>
-  </div>
-</Box>
-
 ## Accessibility
 
-### Known accessibility issues (GitHub staff only)
+For enhanced accessibility, it's important to use explicit saving mechanisms in the select panel. This includes incorporating "Apply" and "Cancel" buttons for both single- and multi-selection scenarios. These buttons provide clear, accessible actions for users to confirm or revoke their selections.
 
- <AccessibilityLink label="SelectPanel"/>
+Automatic saving may be suitable for single selections that do not include additional form controls. In such cases, the selection process is straightforward and does not require further user confirmation, making it accessible and efficient. However, when the selection process involves additional steps or consequences, explicit saving mechanisms are crucial to ensure that all users, including those with accessibility needs, can confidently and accurately manage their selections.
```

Comment:
Random question, for the "instant selection" or simpler cases for selection without a filter, are these situations where we would prefer folks to use components other than a select panel? (like an actionlist or action menu directly)